tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -107,13 +107,17 @@ def run_conv(
         )
 
     tt_input_tensor = ttnn.from_torch(torch_input_tensor, ttnn.bfloat16)
+    if type(use_1d_systolic_array) == ttnn.TensorMemoryLayout:
```

Comment:
Please don't overload variables like this.

Either add another variable that is checked first or rename the variable to e.g. shard_layout

Reply:
Fixed. Added a new variable called shard_layout